modules/monitoring/main.tf

```diff
@@ -0,0 +1,178 @@
+locals {
+  gdb_main_log_table_name        = "gdb_main_log_CL"
```

Comment:
Why upper `CL`? Also, keep in mind that God kills a kitten every time you use `gdb`.
![poor kitten](https://i.pinimg.com/564x/32/f2/43/32f24381b05fcf53d8088c98963fe326.jpg)

Reply:
It comes from azure, some requirement of theirs:
![image](https://github.com/Ontotext-AD/terraform-azure-graphdb/assets/52039166/652392bd-4989-40fa-a7d9-73015d139a63)

As for the usage of gdb, well it's the DB main log, i'm open for suggestions